### DISCUSSION TITLE
Welcome to NLAD Discussions!

### DISCUSSION THREAD

**Discussion by Matt-Dionis** [Matt-Dionis/nlad#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kyle-cassidy** [comment]
thanks for sharing this repo! saw it on /local llama. i gave you a star and i'll check it out!

**Reply by Matt-Dionis** [reply]
Thanks for joining, Kyle! The existing README is just my attempt to get started. I'm hoping to see a lot of community members like yourself share your thoughts, ideas, workflows, etc! 🦾
